### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by Allen-B1**
Added language attributes and description

### CODE REVIEW

File: src/assets/manifest.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
-	"name": "Rebble Store",
+	"name": "Rebble App Store",
+  "short_name": "Rebble Store",
+  "description": "Browse rebble apps on the rebble app store.",
```
Comment (sGerli) [inline src/assets/manifest.json]: The description should be "Browse Pebble apps in the Rebble Store."

**Review by sGerli — CHANGES_REQUESTED** [review]

File: src/assets/manifest.json (hunk 1 above)
Comment (Allen-B1) [reply src/assets/manifest.json]: Oh ok

Comment (Allen-B1) [reply src/assets/manifest.json]: How do I change it?

Comment (Allen-B1) [reply src/assets/manifest.json]: I think I changed it

**Comment by KonradIT** [comment]
Right now should be good to merge

**Pull request merged**